### PR TITLE
Fix avatar styles

### PR DESCRIPTION
**Pull request by tjbo**
https://helpscout.atlassian.net/secure/RapidBoard.jspa?rapidView=153&projectKey=COMMS&modal=detail&selectedIssue=COMMS-330

The user avatars shown on the Chat blank screen are A) blurry and B) have very subtle but noticeable blue pixelation around the image — both of which make them look bad.

This ticket fixes that by adding some styles.

### CODE REVIEW

File: src/components/Avatar/Avatar.Crop.tsx [hunk 1]
```diff
@@ -21,6 +21,7 @@ export const AvatarCrop = props => {
   )
 
   const styles = {
+    color: 'transparent',
```
Comment (knicklabs) [inline src/components/Avatar/Avatar.Crop.tsx]: I think this transparent color needs to be applied only in the case where there is no image. See this example here https://deploy-preview-724--hsds-react.netlify.com/?path=/story/avatar--initials -- the initials are not visible due to the transparent background.

Comment (tjbo) [reply src/components/Avatar/Avatar.Crop.tsx]: Great catch. Fixed. 👍

**Comment by tjbo** [comment]
@digitaldesigner @knicklabs 

Thanks for the feedback. I didn't know Avatars could be so complicated.

Updated the ticket. 👍

File: src/components/Avatar/styles/Avatar.css.ts [hunk 2]
```diff
@@ -94,7 +94,8 @@ export const ActionUI = styled('div')`
 
 export const CropUI = styled('div')`
   align-items: center;
-  background-color: currentColor;
+  background-color: ${({ hasImage }) =>
+    hasImage ? 'transparent' : 'currentColor'};
```
Comment (knicklabs) [inline src/components/Avatar/styles/Avatar.css.ts]: Nice use of props here!

**Pull request merged**